hibike/devices/YogiBear/ui.cpp

```diff
@@ -0,0 +1,173 @@
+#include "Arduino.h" //to get digital write, etc.
+
+#include "ui.h"
+#include "motor.h"
+#include "pid.h"
+#include "encoder.h"
+#include "current_limit.h"
+//Code here that helps deal with the human-readable UI.
```

Comment:
In the end we removed this file and the manual controls. Since it will never be used once the YogiBears have been shipped, and since temporarily recreating some test code will be quick and not a messy as this large set of commands.